### ISSUE TITLE
Error importing NgxKjuaModule on Angular 7

### ISSUE THREAD

**Issue by DanRic** [werthdavid/ngx-kjua#5]
Hi,
when I import NgxKjuaModule 
`import { NgxKjuaModule } from 'ngx-kjua';`
I have this error:
```
ERROR in ./node_modules/ngx-kjua/esm5/ngx-kjua.js
Module not found: Error: Can't resolve 'kjua' in 'C:\....\node_modules\ngx-kjua\esm5'
```

**Comment by werthdavid** [comment]
Have you installed kjua as well?

**Comment by werthdavid** [comment]
should be fixed with 1.3.1
